### DISCUSSION TITLE
Typescript, immer + devtool + persist for slices separately?

### DISCUSSION THREAD

**Discussion by Toshinaki** [pmndrs/zustand#1307] [Q&A] [answered]
I'm migrating my redux app to zustand. My store contains several slices and some need to access state from other slices.
```
{
  user: {...},
  settings: {...},
  theme: {...}
}
```

I searched the docs and questions, but none match my needs.

I want to persist part of the user state and part of the theme state, saved at localstorage with name "saved-user" and "saved-theme".
And in some methods of user, I want to have access to settings state value.

Is there any complex enough example implementing something like this?

**Comment by dai-shi** [comment, marked as answer]
> I want to persist part of the user state and part of the theme state, saved at localstorage with name "saved-user" and "saved-theme".

Sounds like multiple stores would do it pretty easily and straightforward.
